src/Service/AuthenticationService.php

```diff
@@ -44,15 +44,12 @@ public function getLoggedUser(): ?User
         if (!isset($_SESSION['id'])) {
             return null;
         }
```

Comment:
use the session service (inject it in this service), do not use globals directly.